StreamChatUITestsAppUITests/MockResponses/http_message_ephemeral.json

```diff
@@ -0,0 +1,129 @@
+{
+  "message": {
+    "id": "83CA6796-199D-41DB-A6BB-AB063B6514A7",
+    "text": "/giphy /test",
```

Comment:
Does the backend recognize this as a valid giphy command? Can we use `/giphy test`? In the current shape it looks like a message with 2 commands

Reply:
I see 👍 That's a nit but `/giphy /test` text feels weird, tbh. When I first saw the JSON, I thought it was the one to test the `invalid command` behavior

Reply:
yeah, I felt the same, but ... you can't take words out of a song 😄